### DISCUSSION TITLE
Useage of policy in contract definition

### DISCUSSION THREAD

**Discussion by cwBMW** [eclipse-tractusx/ssi-docu#10] [Q&A] [answered]
Hi, 

short question. To use the policy in a contract definition i would then use the id of the offer? Is this correct e.g.: 
payload Policy definition 
```json
 {
  "@context": [
    "https://www.w3.org/ns/odrl.jsonld",
    {
      "cx": "https://w3id.org/catenax/v0.0.1/ns/"
    }
  ],
  "@type": "Offer",
  "@id": "a343fcbf-99fc-4ce8-8e9b-148c97605aab"
......
```
### Payload Contract definition
```json
{
    "id": "9a8f5fff-e7b1-424a-8061-7b57c2bb7c18",
    "accessPolicyId" : "a343fcbf-99fc-4ce8-8e9b-148c97605aab",
    "contractPolicyId" : "a343fcbf-99fc-4ce8-8e9b-148c97605aab",
    "criteria": [
        {
            "operandLeft": "asset:prop:id",
            "operator": "eq",
            "operandRight": "urn:uuid:2fb5b753-551a-4dcc-958c-597bb4cd5869-urn:uuid:60a46f7f-9ea2-4a5b-8ef0-79fae2a43g7z"
        }
    ]
}
```
https://github.com/eclipse-tractusx/ssi-docu/blame/c75f5b9141e00ec6e58c9fc88684fb95f0ba1d63/docs/architecture/cx-3-2/edc/policy.definitions.md#L248

**Comment by jimmarino** [comment]
There should be examples in the documentation. @saschaisele-zf @yurimssilva can you help?

**Comment by saschaisele-zf** [comment, marked as answer]
That is correct.
You can take a look at either the[ postman ](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/docs/development/postman) collection or this [documentation](https://github.com/eclipse-tractusx/tractusx-edc/pull/441/files#diff-020601ff35b3a8ebfa952e8fd0029dba66fe3504beba000f8d722a8c0a332748) about the management API.
